pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -1264,7 +1265,16 @@ def _isingxx_to_cnot_rx_cnot(phi: TensorLike, wires: WiresLike, **__):
     qml.CNOT(wires=wires)
 
 
-add_decomps(IsingXX, _isingxx_to_cnot_rx_cnot)
+def _issingxx_to_ppr_resource():
```

Comment:
Typo: issing -> ising
```suggestion
def _isingxx_to_ppr_resource():
```